karrot/invitations/tests/test_api.py

```diff
@@ -164,6 +164,13 @@ def test_invite_as_newcomer_fails(self):
         response = self.client.post(base_url, {'email': 'please@join.com', 'group': self.group2.id})
         self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
 
+    def test_invite_with_different_invited_by_language(self):
+        self.member.language = 'fr'
+        self.member.save()
+        self.client.force_login(self.member)
+        response = self.client.post(base_url, {'email': 'please@join.com', 'group': self.group.id})
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
```

Comment:
@lenniezelk 
Could you add an assertion that the email is actually in Swedish (or not in English, as that would be the default)?
You should be able to access the subject via `mail.outbox[0].subject`, for example.

Otherwise looks great!